docs/docs-content/clusters/edge/site-deployment/deploy-custom-registries/migrate-edge-native-config.md

```diff
@@ -0,0 +1,73 @@
+---
+sidebar_label: "Migrate from Harbor Edge-Native Config Pack"
+title: "Migrate from Harbor Edge-Native Config Pack"
+description:
+  "This page teaches you how to migrate from the Harbor Edge-Native Config Pack to the Registry Connect pack. "
+hide_table_of_contents: false
+sidebar_position: 80
+tags: ["edge"]
+---
+
+Disconnected Edge clusters are required to use a in-cluster primary registry to store images needed for cluster
+deployment. Previously, this was implemented using the **Harbor Edge-Native Config** pack, which installs the Harbor
+registry as well as handles image pull redirects.
+
+The **Harbor Edge-Native Config** has been deprecated and will no longer be supported in a future release. If you want
+to keep your cluster's registry infrastructure up-to-date, you need migrate off of the legacy pack and use the new
+**Registry Connect** pack to implement the Harbor registry. This allows you to keep using the Harbor registry as the
+primary registry and keep it up-to-date with the latest upstream updates.
+
+## Limitations
+
+- When you migrate off of the **Harbor Edge-Native Config** pack, you cannot change the Harbor registry into another
+  registry. You must select the Harbor registry to use as the primary registry for your cluster.
+
+## Prerequisite
+
+- An existing Edge cluster with the **Harbor Edge-Native Config** pack.
+
+## Migrate from Harbor Edge-Native Config Pack to Registry Connect
+
+1. Log in to [Palette](https:/console.spectrocloud.com).
+
+2. From the left **Main Menu**, click **Profiles**.
+
+3. Select the profile you use to deploy your cluster.
+
+4. Remove the **Harbor Edge-Native Config** pack from the profile.
+
+5. Click **Add New Pack** and select the <VersionedLink text="Harbor" url="/integrations/packs/?pack=harbor" /> pack.
+
+6. In the **values.yaml**, update `charts.harbor.harborAdminPassword` to the _same password_ as your existing Harbor
+   registry.
+
+7. If you have previously changed the serving port, PVC, or TLS certificates used by Harbor, you must modify new Harbor
+   pack parameters to match your previous configurations. You update node port, TLS certificates, and PVC configuration
+   using the following parameters.
+
+   - `charts.harbor.expose.nodePort`
+   - `charts.harbor.expose.tls`
+   - `charts.harbor.persistence.persistentVolumeClaim`
```

Comment:
@kirankn8 does user need to get current password and update here?

Reply:
@kirankn8 Can you help address this question?

Reply:
yes user's need to get the current harbor password and update it here